### DISCUSSION TITLE
How to create a spatial structure‌ ?

### DISCUSSION THREAD

**Discussion by JiaojiaoChen123** [Open-Systems-Pharmacology/Forum#1400] [Questions & Problems]

**Comment by Yuri05** [comment]
please check the published model: https://github.com/Open-Systems-Pharmacology/Dabigatran-Reversal-Model

**Comment by JiaojiaoChen123** [comment]
Thank you for your reply. I checked the published model: https://github.com/Open-Systems-Pharmacology/Dabigatran-Reversal-Model before the consultation. I added "virtual organs" and "Events" referring to their model, but there was no change in plasma drug concentration compared with those before these operations. Could you check my model？
